### ISSUE TITLE
Where is hyperranges.json

### ISSUE THREAD

**Issue by albertchristian92** [grantsrb/PyTorch-A2C#3]
Hello, actually I am interested with your works, however I cannot find the file namely hyperranges.json for training the model. May I know what the file looks like? 

Thank you

**Comment by grantsrb** [comment]
Hi Albert,

The hyperranges json is just a json in which the keys are the hyperparams
you want to search over and the values are a list of values you want to
search over. For example, if you wanted to search over both the learning
rate and the weight decay you would make a json file that holds the
following:

{
  lr: [0.001, 0.0001],
  l2: [0.001, 0.01]
}

I hope this helps!

Best,
Satchel

PS No guarantees the repo will work as it is. I haven't touched it in a
while and I don't have time to debug right now. There are plenty of good
implementations out there if mine doesn't work :)
